presidio-analyzer/pyproject.toml

```diff
@@ -36,9 +36,10 @@ azure-ai-textanalytics = { version = "*", optional = true }
 azure-core = { version = "*", optional = true }
 transformers = { version = "*", optional = true }
 huggingface_hub = { version = "*", optional = true }
+gunicorn = {version = "*", optional = true}
 
 [tool.poetry.extras]
-server = ["flask"]
+server = ["flask", "gunicorn"]
```

Comment:
Please add the `gunicorn` license into the NOTICE file.

Reply:
Updated the NOTICE file, thanks